crt_portal/cts_forms/templates/forms/base.html

```diff
@@ -99,8 +104,10 @@
     </div>
     {% endblock %}
   </body>
+  {% compress js %}
   <script src="{% static 'js/url_params_polyfill.js' %}"></script>
   <script src="{% static 'js/uswds.min.js' %}"></script>
   <script src="{% static 'js/focus_alert.js' %}"></script>
   {% block page_js %}{% endblock %}
```

Comment:
We've got some external sources in templates that use this `page_js` block that end up leading to an exception when testing locally -- touchpoints script in `report_primary_complaint.html` and `confirmation.html` specifically

```
UncompressableFileError at /report/
'https://touchpoints.app.cloud.gov/touchpoints/e0517981/js' isn't accessible via COMPRESS_URL ('/static/') and can't be compressed
```

Reply:
Gosh, great catch, thank you!!

Reply:
Fixed + rebased on dev